.pre-commit-config.yaml

```diff
@@ -20,3 +20,8 @@ repos:
     hooks:
       - id: end-of-file-fixer
       - id: trailing-whitespace
+
+  - repo: https://github.com/pre-commit/pre-commit
+    rev: v3.2.2
+    hooks:
+      - id: validate_manifest
```

Comment:
Curious question: what does this hook do?  I know it's an "official" pre-commit hook, but when I tried to search for its documentation, I couldn't find any.

Thanks!

Reply:
That hook [lints the hook manifest itself](https://github.com/pre-commit/pre-commit/blob/c716de12f7533dc8ce522afb0c09f22476243f08/.pre-commit-hooks.yaml#L3).
So if a repo defines a hook that hook should also be checked to be valid, especially considering future compatibility.

Reply:
Ah I see.  Thank for the explanation!